ethereum/spec/src/main/java/tech/pegasys/teku/spec/Spec.java

```diff
@@ -664,7 +668,9 @@ public BeaconState processBlock(
               blockSlotState,
               IndexedAttestationCache.NOOP,
               signatureVerifier,
-              payloadExecutor);
+              payloadExecutor,
```

Comment:
I don't like passing here these 3 which is not needed on early forks, but we create `payloadExecutor` and `blobsSidecarAvailabilityChecker` for every block, so we could not put it, say, in SpecLogic. We could put there only `kzgCommitmentsProcessor`, which is singleton. We could create very specific container, but it's definitely a bad way. Ok, let it go ;)